src/Uno.UWP/ApplicationModel/Calls/PhoneCallManager.Android.cs

```diff
@@ -0,0 +1,70 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.Content;
+using Android.OS;
+using Android.Telephony;
+using Uno.UI;
+
+namespace Windows.ApplicationModel.Calls
+{
+	public partial class PhoneCallManager
+	{
+		private static readonly TelephonyManager _telephonyManager;
+
+		static PhoneCallManager()
+		{
+			_telephonyManager = (TelephonyManager)ContextHelper.Current
```

Comment:
ContextHelper.Current may be null if this is called too early. Might be good to throw an informative exception in that case.